### DISCUSSION TITLE
Just want an app to launch shell scripts

### DISCUSSION THREAD

**Discussion by jidanni** [termux/termux-app#3182] [Q&A] [answered]
I just want a way to launch a sh script on non-root Android that will keep on working past version 12 and would be able compatible with Google Play rules. I.e., I want to install it from Google Play.

Okay, no more files permission. Well certainly the app could ask the user which script to upload and then keep the copy  in its own area.

Okay no more access to the system's Korn shell. Well I guess the app could have its own built-in shell?

**Comment by sylirre** [comment, marked as answer]
Try ConnectBot if you need an app that comply with Google Play and is not "overloaded". It supports local terminal using utilities provided by Android and SSH or Telnet protocols for remote.

Access to system Korn shell (`/system/bin/sh`) unlikely to be restricted.
